tests/accessor/accessor_api_image_common.h

```diff
@@ -984,10 +984,23 @@ class image_accessor_api_sampled_r {
        *    ftp://ftp.icsi.berkeley.edu/pub/theory/priest-thesis.ps.Z
        *  for details
        */
-      return get_expected_value_linear(idx + 1);
+      return get_expected_value_linear(next_idx(idx));
     }
   }
 
+  template <int dims>
+  sycl::id<dims> next_idx(sycl::id<dims> idx) const {
```

Comment:
```suggestion
  sycl::id<dims> next_id(sycl::id<dims> idx) const {
```
I think it's worth adding a comment clarifying what this function does and why.

Reply:
Thank you for the suggestion, I've renamed the method and added a comment why this method is required.